src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -1,65 +1,66 @@
-import React, { useCallback, useMemo, useState } from "react";
-import { Button, Nav } from "react-bootstrap";
-import { Link } from "react-router-dom";
+import React, { useMemo, useState } from "react";
+import { Nav } from "react-bootstrap";
 import classNames from "classnames";
 
-import AdvancedSettings from "pages/Admin/AdvancedSettings";
-import BasicInfo from "pages/Admin/BasicInfo";
-import EntranceExperience from "pages/Admin/EntranceExperience";
-import VenueDetails from "pages/Admin/Venue/Details";
+import { useVenueId } from "hooks/useVenueId";
+import { useConnectCurrentVenueNG } from "hooks/useConnectCurrentVenueNG";
+import { useUser } from "hooks/useUser";
+import { useRoles } from "hooks/useRoles";
+import { useIsAdminUser } from "hooks/roles";
 
-import { Venue_v2 } from "types/venues";
+import { LoadingPage } from "components/molecules/LoadingPage";
 
 import "./AdminVenueView.scss";
 
 export enum AdminVenueTab {
-  dashboard = "dashboard",
-  basicInfo = "basic_info",
-  entranceExperience = "entrance_experience",
-  advancedMapSettings = "advanced_map_settings",
+  spaces = "spaces",
+  timing = "timing",
+  run = "run",
 }
 
-const adminVenueTabs: Record<AdminVenueTab, String> = {
-  [AdminVenueTab.basicInfo]: "Start",
-  [AdminVenueTab.entranceExperience]: "Entrance",
-  [AdminVenueTab.advancedMapSettings]: "Advanced",
-  [AdminVenueTab.dashboard]: "Dashboard",
+const adminVenueTabs: Readonly<Record<AdminVenueTab, String>> = {
+  [AdminVenueTab.spaces]: "Spaces",
+  [AdminVenueTab.timing]: "Timing",
+  [AdminVenueTab.run]: "Run",
 };
 
-const DEFAULT_TAB = AdminVenueTab.dashboard;
+const DEFAULT_TAB = AdminVenueTab.spaces;
 
-export interface AdminVenueViewProps {
-  venue: Venue_v2;
-}
-
-export const AdminVenueView: React.FC<AdminVenueViewProps> = ({ venue }) => {
+export const AdminVenueView: React.FC = () => {
+  const venueId = useVenueId();
   const [selectedTab, setSelectedTab] = useState<string>(DEFAULT_TAB);
 
-  const selectDefaultTab = useCallback(() => {
-    setSelectedTab(DEFAULT_TAB);
-  }, []);
+  const { user } = useUser();
+  const { roles } = useRoles();
+  const { isAdminUser } = useIsAdminUser(user?.uid);
+
+  const { currentVenue: venue } = useConnectCurrentVenueNG(venueId);
 
   const renderAdminVenueTabs = useMemo(() => {
-    return Object.entries(adminVenueTabs).map(([id, text]) => (
+    return Object.entries(adminVenueTabs).map(([key, text]) => (
       <Nav.Link
-        key={id}
+        key={key}
         className={classNames("AdminVenueView__tab", {
-          "AdminVenueView__tab--selected": selectedTab === id,
+          "AdminVenueView__tab--selected": selectedTab === key,
         })}
-        eventKey={id}
+        eventKey={key}
       >
         {text}
       </Nav.Link>
     ));
   }, [selectedTab]);
 
+  if ((venueId && !venue) || !roles) {
+    return <LoadingPage />;
+  }
+
+  if (!roles.includes("admin") || !isAdminUser) {
```

Comment:
Why does it doublechecks if a user is an admin?

Reply:
There are 2 places which contain admin user ids. The `users` collection, and the venue field `owners`. This checks both places.

Reply:
I don't think it's the case. Looking at the way the hooks are implemented, neither of them checks for venue ownership

![image](https://user-images.githubusercontent.com/63194656/121515181-9346e700-c9f5-11eb-9df5-ccbae9208aee.png)

Reply:
You are right, `owners` is being used only to display the list for venues in the admin screen when you open it. 

In this case `useRoles` is unnecessary, I will remove it and use `useIsAdminUser`

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1511/commits/d74ce15ca534e3215cda469caf147a7a6d26e9d5